ci/code_checks.sh

```diff
@@ -24,7 +24,7 @@ CHECK=$1
 
 
 ### LINTING ###
-if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
+if [[ -z "$CHECK" || "$CHECK/lint" == "lint" ]]; then
```

Comment:
I may be missing something, but I think the equality with this will compare `"/lint" == "lint"` or `"lint/lint" == "lint"` and will be always false.